config/config.py

```diff
@@ -1,4 +1,5 @@
 import os
 
```

Comment:
is this file needed ?

Reply:
yes. It makes it easier to update when configs change